batch/test/test_accounts.py

```diff
@@ -398,3 +398,101 @@ async def test_billing_limit_tiny(make_client, dev_client, new_billing_project):
     batch = await batch.submit()
     batch = await batch.wait()
     assert batch['state'] == 'cancelled', batch
+
+
+async def test_all_billing_project_users_operations(make_client, dev_client, new_billing_project):
```

Comment:
and this is something like `test_user_can_access_batch_made_by_other_user_in_shared_billing_project`, yeah?